pages/index.js

```diff
@@ -1,7 +1,106 @@
-export default function HomePage() {
-	return (
-		<>
-			<h1>My Cosmetics Store</h1>
-		</>
-	);
+import styled from "styled-components";
+
+import { Btn } from "@/components/Button";
+
+import React, { useState } from "react";
+
+import myComponent from "@/components/myComponent1/myComponent.js";
```

Comment:
Either this 
```suggestion
import Title from "@/components/Title.js";
```

or see the comment in "components/myComponent1/myComponent.js"